scripts/components/SongCards.js

```diff
@@ -101,7 +101,7 @@ class SongCards extends Component {
 
     renderSongs(start, end) {
         const chunk = 5;
-        const {authed, dispatch, playlist, playlists, playingSongId, songs, users} = this.props;
+        const {authed, dispatch, playlist, playlists, playingSongId, songs, users, toggleStats} = this.props;
```

Comment:
There a few different ways to go about it. I know a few people who feel hate the idea of using `mapStateToProps` since it's somewhat akin to "monkey patching", but when you have a large app with a huge number of props, i feel like it makes sense to use `mapStateToProps` to make group related props to make your app's data easier to reason about.

Also, there's somewhat of a performance boost since it cuts down on the total number of chained `render()`'s being called.